.buildscript/deploy_snapshot.sh

```diff
@@ -1,26 +0,0 @@
-#!/bin/bash
```

Comment:
This is Travis-specific and not needed anymore, `.github/workflows/build.yml` checks preconditions and `./gradlew uploadArchives` is enough to publish.